### ISSUE TITLE
It displays Arabic-Indic numbers when use react-intl with 'ar' in FireFox.

### ISSUE THREAD

**Issue by SunShaoze** [formatjs/formatjs#4004]
I hope you can see if this is a bug. Should numbers remain unchanged instead of being translated into specific languages?
![image](https://user-images.githubusercontent.com/39686047/220909046-edc57b92-3d13-4aa7-a591-fe3508c2cf38.png)
![image](https://user-images.githubusercontent.com/39686047/220908926-6359d6f4-09e2-401e-8ca7-024b0e47b534.png)

**Comment by ez4gis** [comment]
Hi @SunShaoze, I recently encountered same as well. And would be intersted to learn more about the solution or best practice.

For me, I now explicitly specify `numberingSystem` as either `latn` or `arab` in the option when calling `formatNumber()` in order to enforce consistency between browsers. It will use either western numericals or eastern arabic numericals depending on the numbering system passed in. Worked well for Firefox, Chrome, Edge Chromium, and Safari.  

```js
formatNumber(someVal / 100.0, {
  style: 'percent',
  numberingSystem: 'latn',
  // ...
});
```

However, I did get a feedback that the *percentage sign* should be on the **left** of numbers (for example: `%18`) instead of on the **right**(`18%`) when using western numericals.  I dont know if it's a bug with Intl or react-intl, whether there is a work-around, or if both formats are accepted in arabic language though -- did some googling unfortuantely without a definitive answer on that.
